game/src/layer/mod.rs

```diff
@@ -182,7 +182,7 @@ impl State for PickLayer {
                 }
                 "parking occupancy" => {
                     app.layer = Some(Box::new(parking::Occupancy::new(
-                        ctx, app, true, true, true, false,
+                        ctx, app, true, true, true, false, true,
```

Comment:
I wish Rust allowed named arguments for methods. 😞

Reply:
If this code was called in more places, it'd definitely be time for a struct...